### ISSUE TITLE
error: unable to recognize no matches for kind "RemoteResource" in version "kapitan.razee.io/v1alpha1"

### ISSUE THREAD

**Issue by ericmillin** [razee-io/Razee#64]
This is identical issue to #47 , which was closed.
```
namespace/razee created
deployment.apps/remoteresource-controller created
customresourcedefinition.apiextensions.k8s.io/remoteresources.deploy.razee.io created
serviceaccount/razeedeploy-sa created
clusterrole.rbac.authorization.k8s.io/razeedeploy-admin-cr created
clusterrolebinding.rbac.authorization.k8s.io/razeedeploy-rb created
configmap/razeedeploy-delta-resource-uris created
deployment.apps/razeedeploy-delta created
configmap/watch-keeper-config created
secret/watch-keeper-secret created
Error from server (AlreadyExists): error when creating "blah": namespaces "razee" already exists
unable to recognize "blah": no matches for kind "RemoteResource" in version "kapitan.razee.io/v1alpha1"
```

The problem is `kapitan.razee.io` should be `deploy.razee.io`

**Comment by ericmillin** [comment]
@alewitt2

**Comment by alewitt2** [comment]
@ericmillin where are you installing these resources from. Yes whereever you are installing from, it should be deploy.razee.io

**Comment by ericmillin** [comment]
`ibm.razee.io`

**Comment by adamkingit** [comment]
Code was fixed with issue #47 but new version not yet rolled out
